docs/source/_templates/autosummary/class.rst

```diff
@@ -1,11 +1,12 @@
-gstools.covmodel.base
----------------------
+{{ fullname | escape | underline}}
 
-.. automodule:: gstools.covmodel.base
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
    :members:
    :undoc-members:
-   :show-inheritance:
    :inherited-members:
+   :show-inheritance:
 
 .. raw:: latex
 
```

Comment:
Sometimes git tries to do its best to interpret changes but fails ungracefully.

Reply:
What a git!